ui/docs/tutorial/debug-task.md

```diff
@@ -16,7 +16,7 @@ features, debug with Taskcluster.
 You can run a task on any computer with Docker installed.
 
 First, navigate to your task in the [task
-inspector](https://tools.taskcluster.net/groups) and select the "task
+inspector](https://tc.example.com/groups) and select the "task
```

Comment:
Let's just remove all of the tools links in the tutorial and leave the text un-linked.  As a group, we kind of hate the tutorial and want to rewrite it from scratch, so putting much work into it probably isn't worthwhile.

Reply:
Gotcha.